src/css/profile/mobile/common/button.less

```diff
@@ -675,3 +625,52 @@ a.ui-btn {
 
 	}
 }
+
+// Common classes for icons, not only for buttons
```

Comment:
How about creating the common.less or icons.less?

Reply:
I thought the same :) ... I'll move this to icons.less

Reply:
Done.